### ISSUE TITLE
Iperf testing with WAN acceleration not showing performance increase.

### ISSUE THREAD

**Issue by fplessius** [esnet/iperf#326]
Dear Iperf team,

When we are testing with Iperf in a network with WAN accelerators e.g. Cisco WAAS, Riverbed and others the test results do not show acceleration performance. It seems the IP/TCP payload is not repeating or is a constant. Therefore the accelerators like, LZ compression and Data Redundancy Elimination (DRE) are unable to be tested and performance increase end to end is not detected.
I would like to know if repeating payloads can be added as a feature into Iperf v3?

High overview how DRE is working on Cisco WAN accelerators: http://www.cisco.com/c/en/us/products/collateral/routers/wide-area-application-services-waas-software/white_paper_c11-676350.html 
(the web-page is also valid for the new OS releases)

Regards,
Frits Plessius

**Comment by bmah888** [comment]
What about creating a file on the client whose content has the desired characteristics and then using the ```-F``` option to send that file?

**Comment by fplessius** [comment]
Indeed we could do this but we mostly we do not want to create files on servers that we do not maintain.
We only request to the end user to add the application and then we run iperf remotely.
Therefore was looking for a possible solution to have a repeat pattern in Iperf itself.

**Comment by TheRealDJ** [comment]
If you use '-F' on the client side but not the server side, no files will be created on the server side. Read the man page.

**Comment by bmah888** [comment]
Closing.  I think there's a solution in the last comment.

**Comment by fplessius** [comment]
Dear Mr Mah,

My apologies for not replying to the forum.
The option –F was not doable for us as we needed to have repeating patterns in the data transfers between our customer servers, which Verizon does not own.
Therefore the –F option was not valid for us to use.
We have purchased the Ixia Chariot package on which we could test between the software clients whom are controlled and report their data to a Verizon master server.

Regards,
Frits Plessius
